### PR TITLE
MeshMap page section (v2, accordion)

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR adds a new section in meshmap page (with accordion effect)

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shivam-AfA** [comment]
@leecalcote You can see how the accordion thing would look like on this preview. Working on the scrolling.

**Comment by l5io** [comment]
🚀 Preview for commit 0f411f7b771786b0859cb97a770f0f2e4d422746 at: https://63ed346d4f62710b63b5b0ea--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA how are you doing on my final feedback?

**Comment by l5io** [comment]
🚀 Preview for commit d81ba80982099b16199f88bfc210b9ab2d188b8d at: https://63ef15446ab7ad27479faf63--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7b97d55d1f604506194563eaee6fed001c5d49f0 at: https://63ef277d51f2f731ab458fbc--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
@leecalcote Do you want the context aware transition in place of the designer ones? Then we would need atleast 4 points in the designer section to correctly show the transitions.

**Comment by Shivam-AfA** [comment]
Oh no.. there's something wrong...

**Comment by leecalcote** [comment]
> 

The Collaboration SVGs need replaced.

**Comment by l5io** [comment]
🚀 Preview for commit 6592c0270603fd97a8555e84babaa4abc34520ce at: https://63ef8c9c437ff10af61c48a0--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2a4a51d989e082a6728a44983e4cb0d24ffb38bb at: https://63ef977b8b8cda11702b4423--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 2bfd583ce97aa612855d05db0eee9c41b2217886 at: https://63efa1c46cb659006c2b34a4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 40199e533682273a3a4c924e5c3ea6fe3be5f34a at: https://63eff9da38cdc30b24bb54ba--layer5.netlify.app

**Comment by Chadha93** [comment]
@Shivam-AfA I think we should avoid containerizing the whole page.

**Comment by Chadha93** [comment]
Lot's of commented code, make sure to remove it in the final commit.

**Comment by Shivam-AfA** [comment]
> Lot's of commented code, make sure to remove it in the final commit.

Yes. I'll remove it.

**Comment by Shivam-AfA** [comment]
> @Shivam-AfA I think we should avoid containerizing the whole page.

Like having separate containers for individual sections?

**Comment by l5io** [comment]
🚀 Preview for commit 5872e1bbfafc8fabcb31097a3377884ec4bd93b3 at: https://63f1565deeaea565c78d22a6--layer5.netlify.app

File: src/sections/Meshmap/index.js [hunk 1]
```diff
@@ -75,6 +75,10 @@ const Meshmap = () => {
                   title: "Where all your design assets call home",
                   description: ""
```
Comment (Shivam-AfA) [inline src/sections/Meshmap/index.js:76]: @leecalcote Two points required here.

**Comment by Shivam-AfA** [comment]
@leecalcote In the collaborate section also, we showcase 4 SVG transitions, but have 5 points listed.

**Comment by leecalcote** [comment]
> @leecalcote In the collaborate section also, we showcase 4 SVG transitions, but have 5 points listed.

Ok, thanks for pointing this out. No sweat. I'll update.

**Comment by l5io** [comment]
🚀 Preview for commit 635fb26f94dc105618097191a9e7f81093b9a4db at: https://63f15e6b6938f5600455d5e9--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9adc5454877edace87186c09e0f0f29257c8acb3 at: https://63f1656039e955712e3f5812--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 691b4f39df978bc62658d94371b19337b261a143 at: https://63f169f0c155447157e6028d--layer5.netlify.app

**Pull request merged**